torch/cpu/__init__.py

```diff
@@ -171,9 +171,9 @@ def set_device(device: _device_t) -> None:
     """
 
 
-def current_device() -> str:
-    r"""Returns current device for cpu. Always 'cpu'.
+def current_device() -> int:
+    r"""Returns current device for cpu. Always 0.
```

Comment:
I wonder if this would have backward compatibility issue.
Cc @albanD @janeyx99 to comment

Reply:
Yea this is 100% not backwards compatible!! We should not do this without the proper deprecation cycle, and before then, the reason that this then "lines up with cuda" is not sufficient to go through with this.